release.sh

```diff
@@ -2,29 +2,11 @@
 
 set -euo pipefail
 
-VERSION="${1:-$(echo "${YB_GIT_BRANCH:-}" | sed -e 's:^refs/tags/::')}"
-if [[ -z "$VERSION" ]]; then
-  echo "release.sh: no version provided" 1>&2
-  exit 1
-fi
-
-if $(echo $VERSION | grep -vqo '^v'); then
-    echo "Doesn't start with a \"v\" when it should, not releasing"
-    exit 1
-fi
-
-if $(echo $VERSION | grep -qo '\-[a-z]\+[0-9]*$'); then
-    # Release candidate release
-    CHANNEL="preview"
-else
-    # Stable releases should not have a suffix
-    CHANNEL="stable"
-fi
+. variables.sh
```

Comment:
nit: expand out `source` for clarity.

```suggestion
source variables.sh
```